### ISSUE TITLE
'loss': 0.0

### ISSUE THREAD

**Issue by Purshow** [OpenGVLab/InternVL#655]
请问下大佬loss恒为0是怎么回事？
set -x

OUTPUT_DIR=${1}
GPUS=${2}
BATCH_SIZE=${BATCH_SIZE:-96}
PER_DEVICE_BATCH_SIZE=$((BATCH_SIZE / GPUS))
GRADIENT_ACC=1

python -u internvl_chat_pretrain.py \
  --vision_path "clip-vit-large-patch14-336" \
  --llm_path "internlm2-chat-1_8b" \
  --conv_style "internlm2-chat" \
  --output_dir ${OUTPUT_DIR} \
  --meta_path "data_llava_pretrain.json" \
  --overwrite_output_dir True \
  --force_image_size 336 \
  --max_dynamic_patch 12 \
  --down_sample_ratio 0.5 \
  --drop_path_rate 0.1 \
  --freeze_llm True \
  --freeze_mlp False \
  --freeze_backbone True \
  --vision_select_layer -1 \
  --dataloader_num_workers 4 \
  --bf16 True \
  --num_train_epochs 1 \
  --per_device_train_batch_size ${PER_DEVICE_BATCH_SIZE} \
  --gradient_accumulation_steps ${GRADIENT_ACC} \
  --evaluation_strategy "no" \
  --save_strategy "steps" \
  --save_steps 200 \
  --save_total_limit 3 \
  --learning_rate 2e-5 \
  --weight_decay 0.01 \
  --warmup_steps 100 \
  --lr_scheduler_type "cosine" \
  --logging_steps 1 \
  --max_seq_length 4096 \
  --do_train True \
  --grad_checkpoint True \
  --group_by_length False \
  --dynamic_image_size False \
  --use_thumbnail True \
  --ps_version 'v2' \
  --deepspeed "zero_stage1_config.json" \
  --report_to "tensorboard" \
  2>&1 | tee -a "${OUTPUT_DIR}/training_log.txt"

**Comment by awakenlee180** [comment]
我也有这个问题，loss是0，或许只是print出来的问题？

**Comment by awakenlee180** [comment]
请问您是怎么解决的这个问题？
